tools/check-usage.sh

```
@@ -0,0 +1,282 @@
+    elif (( age_sec < 86400 )); then
+        echo "$(( age_sec / 3600 ))h ago"
+    elif (( age_sec < 604800 )); then
+        echo "$(( age_sec / 86400 ))d ago"
+    else
+        echo "$(( age_sec / 604800 ))w ago"
+    fi
+}
+
+show_monitor() {
+    clear
+    local now_str=$(date +"%Y-%m-%d %H:%M:%S")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)